#! /bin/sh -e

# DP: adds LSB formatted dependency info in init.d script (closes: #469702)

dir=
if [ $# -eq 3 -a "$2" = '-d' ]; then
    pdir="-d $3"
    dir="$3/"
elif [ $# -ne 1 ]; then
    echo >&2 "usage: `basename $0`: -patch|-unpatch [-d <srcdir>]"
    exit 1
fi
case "$1" in
    -patch)
        patch $pdir -f --no-backup-if-mismatch -p1 < $0
        #cd ${dir}gcc && autoconf
        ;;
    -unpatch)
        patch $pdir -f --no-backup-if-mismatch -R -p1 < $0
        #rm ${dir}gcc/configure
        ;;
    *)
	echo >&2 "usage: `basename $0`: -patch|-unpatch [-d <srcdir>]"
        exit 1
esac
exit 0

diff -urNad isdnutils-3.9.20060704~/eurofile/scripts/eftd.sh.in isdnutils-3.9.20060704/eurofile/scripts/eftd.sh.in
--- isdnutils-3.9.20060704~/eurofile/scripts/eftd.sh.in	2008-04-04 11:41:12.000000000 +0200
+++ isdnutils-3.9.20060704/eurofile/scripts/eftd.sh.in	2008-04-04 11:41:58.000000000 +0200
@@ -1,5 +1,12 @@
 #!/bin/sh
 # @configure_input@
+### BEGIN INIT INFO
+# Provides:          isdneurofile
+# Required-Start:    $remote_fs $syslog
+# Required-Stop:     $remote_fs $syslog
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+### END INIT INFO
 #
 # generated from $Id: eftd.sh.in,v 1.8 2002/01/31 18:54:38 paul Exp $
 #
